src/main/java/uk/gov/hmcts/reform/sscs/ccd/exception/AppealNotFoundException.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sscs.ccd.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND, reason = "No appeal for given id")
+public class AppealNotFoundException extends RuntimeException {
```

Comment:
Don't we have to use here the UnknownErrorCodeException for the AppIsight alerts etc?

Reply:
I reckon the reason it was written like this is because we don't want to be alerted every time an appeal cannot be found